split_expenses_ui/src/pages/QuickExpense/QuickExpense.jsx

```diff
@@ -22,28 +30,30 @@ class QuickExpense extends Component {
   };
 
   handleSubmit = async () => {
-    const { header, line_item } = await getProductsFromInvoice(this.state.file);
+    this.setIsLoading();
+    const { header, products } = await getProductsFromInvoice(this.state.file);
 
     let updatedState = this.state;
-    for (let i = 0; i < line_item.length; i++) {
+    for (let i = 0; i < products.length; i++) {
       let data = {};
-      data["productName"] = line_item[i]["Description"];
-      data["price"] = line_item[i]["price"];
-      data["qty"] = line_item[i]["Qty"];
-      data["id"] = i + 1;
+      data["product_name"] = products[i]["description"];
```

Comment:
I can send the product_name value instead of the description name .. so we don't need to map it to a new value. I'll update these changes and will create a new PR